packages/combo-box/src/vaadin-combo-box-scroller.js

```diff
@@ -245,7 +246,14 @@ export class ComboBoxScroller extends PolymerElement {
   /** @private */
   __loadingChanged(loading) {
     if (this.__virtualizer && !loading) {
-      setTimeout(() => this.requestContentUpdate());
+      this.requestContentUpdate();
+    }
+  }
+
+  /** @private */
+  __selectedItemChanged() {
```

Comment:
I had to partially cherry-pick [this fix](https://github.com/vaadin/web-components/pull/4036) too because there were some tests relying on `setTimeout` in `__loadingChanged` that were failing without it.